src/iomei.cpp

```diff
@@ -2455,6 +2455,9 @@ bool MEIInput::IsAllowed(std::string element, Object *filterParent)
         else if (element == "artic") {
             return true;
         }
+        if (element == "verse") {
```

Comment:
Guess this should be:

```suggestion
       else if (element == "verse") {
```

?

Reply:
Yes, thanks for pointing this out!